bundles/org.openhab.binding.netatmo/src/main/java/org/openhab/binding/netatmo/internal/api/data/NetatmoConstants.java

```diff
@@ -0,0 +1,328 @@
+
+                channels.put(String.join("-", apiDescriptor, "measurement"),
+                        new MeasureChannelDetails(apiDescriptor, String.join(":", CoreItemFactory.NUMBER, dimension),
+                                String.format("%%.%df %s", measureDefinition.scale, UnitUtils.UNIT_PLACEHOLDER)));
+                if (canScale) {
+                    channels.put(String.join("-", apiDescriptor, GROUP_TIMESTAMP), new MeasureChannelDetails(
+                            GROUP_TIMESTAMP, CoreItemFactory.DATETIME, "%1$tA, %1$td.%1$tm. %1$tH:%1$tM"));
+                }
+            }
+        }
+    }
+
+    // Netatmo API urls
+    public static final String URL_API = "https://api.netatmo.com/";
+    public static final String URL_APP = "https://app.netatmo.net/";
+    public static final String PATH_OAUTH = "oauth2/token";
+    public static final String PATH_API = "api";
+    public static final String PATH_COMMAND = "command";
+    public static final String SPATH_PERSON_AWAY = "setpersonsaway";
+    public static final String SPATH_PERSON_HOME = "setpersonshome";
+    public static final String SPATH_HOMES = "homesdata";
+    public static final String SPATH_GETHOME = "gethomedata";
+    public static final String SPATH_GETCAMERAPICTURE = "getcamerapicture";
+    public static final String SPATH_ADDWEBHOOK = "addwebhook";
+    public static final String SPATH_DROPWEBHOOK = "dropwebhook";
+    public static final String SPATH_SETROOMTHERMPOINT = "setroomthermpoint";
+    public static final String SPATH_SETTHERMMODE = "setthermmode";
+    public static final String SPATH_SWITCHSCHEDULE = "switchschedule";
+    public static final String SPATH_GETTHERMOSTAT = "getthermostatsdata";
+    public static final String SPATH_GETSTATION = "getstationsdata";
+    public static final String SPATH_GETMEASURE = "getmeasure";
+    public static final String SPATH_HOMESTATUS = "homestatus";
+    public static final String SPATH_HOMECOACH = "gethomecoachsdata";
+    public static final String SPATH_GETLASTEVENT = "getlasteventof";
+    public static final String SPATH_PING = "ping";
+
+    public static final String PARM_DEVICEID = "device_id";
+    public static final String PARM_MODULEID = "module_id";
+    public static final String PARM_HOMEID = "home_id";
+    public static final String PARM_ROOMID = "room_id";
+    public static final String PARM_PERSONID = "person_id";
+    public static final String PARM_SCHEDULEID = "schedule_id";
+    public static final String PARM_DEVICETYPE = "device_types";
+    public static final String PARM_GATEWAYTYPE = "gateway_types";
+    public static final String PARM_MODE = "mode";
+    public static final String PARM_URL = "url";
+    public static final String PARM_CHANGESTATUS = "changestatus";
+    public static final String PARM_FLOODLIGHTSET = "floodlight_set_config";
+    public static final String PARM_FAVORITES = "get_favorites";
```

Comment:
Did you use PARM deliberately or is it a typo for PARAM ?

Reply:
Well, let me rename these...